src/Eluceo/iCal/Component/Event.php

```diff
@@ -381,13 +391,21 @@ public function setSequence($sequence)
     }
 
     /**
-     * @param $organizer
-     *
+     * @return int
+     */
+    public function getSequence()
+    {
+        return $this->sequence;
+    }
+
+    /**
+     * @param string $name
+     * @param string $email
      * @return $this
      */
-    public function setOrganizer($organizer)
+    public function setOrganizer($name, $email = '')
```

Comment:
I think it is better to pass the Organizer object rather than two strings. I will merge it and then I will change this afterwards.